root/etc/e-smith/templates/etc/pam.d/cockpit/10base

```diff
@@ -0,0 +1,24 @@
+#%PAM-1.0
+auth	   required	pam_sepermit.so
+auth       substack     password-auth
+auth       include      postlogin
+auth       optional     pam_ssh_add.so
+# the root and members of domain admins are always allowed
+# if pam auth succeed, it ignores  pam_listfile.so
+auth [success=2 default=ignore] pam_succeed_if.so user in [root]
+auth [success=1 default=ignore] pam_succeed_if.so user ingroup [domain admins]
+# select group allowed to login to cockpit (root & domain admins alway OK)
+auth    required       pam_listfile.so \
```

Comment:
This module is not always required. If the ShellOverrideStatus prop is disabled, we must comment it to implement the "legacy" behavior. 

I'd prefer to not output it at all, so that our template echoes out the same original PAM config from the cockpit RPM.